nautobot/ipam/tests/test_forms.py

```diff
@@ -9,62 +10,63 @@
 from nautobot.ipam.models import IPAddress, Namespace, Prefix
 
 
-class BaseNetworkFormTest:
-    form_class = None
-    field_name = None
-    object_name = None
-    extra_data = {}
-
-    def setUp(self):
-        super().setUp()
-        self.namespace = Namespace.objects.create(name="IPAM Form Test")
-        self.status = Status.objects.get(name="Active")
-        self.prefix_status = Status.objects.get_for_model(Prefix).first()
-        self.ip_status = Status.objects.get_for_model(IPAddress).first()
-        self.parent = Prefix.objects.create(
-            prefix="192.168.1.0/24", namespace=self.namespace, status=self.prefix_status
-        )
-        self.parent2 = Prefix.objects.create(
-            prefix="192.168.0.0/16", namespace=self.namespace, status=self.prefix_status
-        )
-        self.parent6 = Prefix.objects.create(
-            prefix="2001:0db8::/40", namespace=self.namespace, status=self.prefix_status
-        )
+class NetworkFormTestCases:
+    class BaseNetworkFormTest(TestCase):
```

Comment:
Changed this to a TestCase subclass so that pylint wouldn't complain about it trying to call methods like `self.assertTrue`. Moved it to an inner class (similar to other generic test base classes) so that unittest won't discover and run this base class directly.